README.md

```diff
@@ -1,37 +1,79 @@
-# Shark Cage Team project
+# Shark Cage
 
 [![Build status](https://ci.appveyor.com/api/projects/status/jxhrl395fvtjhwc6?svg=true)](https://ci.appveyor.com/project/SharkCagey/htwg-shark-cage)
 
-This software is provided "as is", without warranty of any kind. This software is also still under development. Use entirely at your own risk.
+This software is provided "as is", without warranty of any kind. This software is also still under development. Use entirely at your own risk. Contributions through PRs are highly appreciated.
 
-This project contains windows programs to isolate a program in an own desktop to prevent malware (without OS privileges) to capture user input or sreenshots.
-It consists of 4 sub-programs: The `CageService`, `CageManager`, `StarterCmd`, and `ImageSelectDialog`. The 3 actual parts (Service, Manager and StarterCmd) communicate over a TCP connection to interchange messages.
+This project contains Windows programs to isolate a user specific program in an own desktop to prevent malware (without OS privileges) to capture user input or screenshots.
+After the installation of Shark Cage, create a config using the `CageConfigurator` containing the program (e.g. Firefox for online banking) which should run in a secure environment, optionally an additional programm (e.g. Keepass to retrieve the password for the online banking) and an icon which is later used to identify, that the program was started in the secure environment.
+
+All programs running in the Shark Cage will be started on a further, isolated desktop which a malware without administrator privilieges can not access.
 
 ## Components
 
+This project consists of five sub-programs:
+
+* `CageService`
+* `CageManager`
+* `CageChooser`
+* `CageConfigurator`
+* `SharkCageInstaller`
+
+ The three parts (Service, Manager and Configurator) communicate over a TCP connection to interchange messages.
+
 ### CageService
 
-The `CageService` implements a Windows service running the background. It receives messages from the UI and sends messages to the `CageManager`.
+The `CageService` implements a Windows service running in the background. It receives messages from the `CageChooser` and sends messages to the `CageManager`.
 
 ### CageManager
 
-The `CageManager` creates a new desktop and processes in it, according to the messages received by the `CageService`.
+The `CageManager` creates a new desktop and starts the program and optionally the additional program according to the config received from the `CageChooser` over the `CageService`. In addition, the icon and some additional informations are displayed. Using the displayed "Activate"-button(s) the program(s) can be restarted or brought back into the foreground.
```

Comment:
'In addition, the icon and some additional informations are displayed.' -> 'In addition, the token image and some additional information is displayed.'